gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -175,6 +177,36 @@ def main(args):
                 ).write(training_resources[region].path, overwrite=overwrite)
             logger.info("Done with creating training dataset.")
 
+        # Build plateau and coverage models for autosomes/pseudoautosomal regions,
+        # chromosome X, and chromosome Y
+        if args.build_models:
+            plateau_models = {}
+            # Check if the training datasets exist.
+            check_resource_existence(
+                input_pipeline_step="--create-training-set",
+                input_resources=training_resources.values(),
+                overwrite=overwrite,
+            )
+            # Build plateau and coverage models.
+            for region in GENOMIC_REGIONS:
+                if region == "autosome_par":
```

Comment:
I let `build_models()` skips the building coverage model part and returns `None` if it's not necessary to build coverage model, but it's a bit redundant using if else statement. Please let me know if there is a better way to deal with it!

Reply:
my thinking was something like this
```
coverage_model, plateau_models["autosome_par"] = build_models(
    training_resources["autosome_par"].ht(),
    weighted=use_weights,
    pops=POPS if use_pops else (),
    build_coverage_model=True,
)

for region in ["chrx_nonpar", "chry_nonpar"]:
    coverage_model, plateau_models[region] = build_models(
        training_resources["autosome_par"].ht(),
        weighted=use_weights,
        pops=POPS if use_pops else (),
    )
```
though this is no less redundant than what you've written :) 

also, have you confirmed that we are only using the autosomal coverage model for v4? this entire discussion might be moot if we're going to calibrate autosomal and allosomal site coverage differently